docker-compose-cluster.yaml

```diff
@@ -308,10 +356,18 @@ services:
       dockerfile: Dockerfile
       args:
       - SENSU_CLI_VERSION
-      - VAULT_VERSION  
+      - VAULT_VERSION
     image: workshop:${SENSU_CLI_VERSION}
+    deploy:
+      resources:
+        limits:
+          memory: 64M
+      restart_policy:
+        condition: on-failure
+        max_attempts: 3
```

Comment:
If I change this to max_attempts: 5   things work on my local system.

Maybe max_attempts: 10  to be safe?

Reply:
Made them configurable :+1: and set the default to 10 😄